app/src/main/java/com/mediaan/mymediaan/view/ftu/CreateProfileScreen.kt

```diff
@@ -176,13 +189,40 @@ fun CreateProfileScreen(
                 maxValue = ftuViewModel.maxCharFacts,
                 labelId = R.string.create_profile_screen_lie_label
             )
+            Text(
+                text = stringResource(id = R.string.create_profile_screen_heading_interests),
+                modifier = Modifier
+                    .padding(top = 24.dp)
+                    .padding(bottom = 16.dp),
+                style = Typography.bodyLarge,
+                fontWeight = FontWeight.SemiBold,
+            )
+            LazyColumn(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(480.dp)
+            ) {
+                items(Interest.entries.toTypedArray()) { interest ->
+                    Row(
+                        verticalAlignment = androidx.compose.ui.Alignment.CenterVertically,
+                        modifier = Modifier
+                            .fillMaxWidth()
+                    ) {
+                        Checkbox(
+                            checked = selectedInterests.contains(interest),
+                            onCheckedChange = { toggleInterest(interest) },
+                            colors = CheckboxDefaults.colors(checkedColor = MediaanPrimary)
+                        )
+                        Text(text = interest.toString(), modifier = Modifier)
+                    }
```

Comment:
I would go for making the whole row clickable. Bigger tap area and more modern UX imo.